app/webpacker/packs/application.js

```diff
@@ -1,38 +1,38 @@
-import "../styles/application.scss";
```

Comment:
Our ruby linter wants double quotes so this will cause confusion...

Reply:
I've been thinking about your comment and at first, I thought it might be a deal-breaker.  StandardJS does have an flag `--fix` you could pass to it to fix issues with double quotes not being single. We could wire that up to the rake task as we do for the current ruby linter. 

Do you think its still a deal-breaker?

Reply:
I just think we should use double quotes in our JS as well as Ruby. Assuming we can change the config for this linter...

Reply:
This is the one of Standard.j

Reply:
Unfortunately, we can't change Standard.js rules 

> The whole point of standard is to save you time by avoiding bikeshedding about code style. There are lots of debates online about tabs vs. spaces, etc. that will never be resolved. These debates just distract from getting stuff done. At the end of the day you have to 'just pick something', and that's the whole philosophy of standard -- its a bunch of sensible 'just pick something' opinions. Hopefully, users see the value in that over defending their own opinions.

> Pro tip: Just use standard and move on. There are actual real problems that you could spend your time solving! :P

https://standardjs.com/#why-should-i-use-javascript-standard-style
https://standardjs.com/#i-disagree-with-rule-x-can-you-change-it
https://standardjs.com/#but-this-isnt-a-real-web-standard